src/config/config.c

```diff
@@ -153,51 +146,61 @@ static int read_main_elements(const OS_XML *xml, int modules,
 #else
             mwarn("%s configuration is only set in the manager.", node[i]->element);
```

Comment:
Remove this. It is not necessary the `ifdef`.

Reply:
Done